hibernate-core/src/main/java/org/hibernate/sql/ast/tree/expression/SqlTupleContainer.java

```diff
@@ -20,4 +20,11 @@ static SqlTuple getSqlTuple(SqlAstNode expression) {
 		}
 		return null;
 	}
+
+	static Expression getExpression(SqlAstNode expression) {
+		if ( expression instanceof SqlTupleContainer ) {
+			return ( (SqlTupleContainer) expression ).getSqlTuple().getExpressions().get( 0 );
```

Comment:
Would be nice if you added `assert ( (SqlTupleContainer) expression ).getSqlTuple().getExpressions().size() == 1;`

Reply:
added